src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
+@import "./utils/variables.scss";
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $column-size;
+  gap: 48px;
 
   max-width: 944px;
-  margin: 0 auto;
+
+  @media (min-width: 488px) and (max-width: 767px) {
+      grid-template-columns: repeat(2, $column-size);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+      grid-template-columns: repeat(3, $column-size);
+  }
+
+  @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, $column-size);
+  }
```

Comment:
Better, but you can remember that in SCSS you can use CSS. So you cam watch 
[PL_FE_MAR23_LESSON 20_03_2023.mp4] from 1:29:27 or try learn more about each https://sass-lang.com/documentation/at-rules/control/each . There you can click always on "=> CSS" and you will see how it compiles to css. IT is very helpful.

Reply:
Good point, but for this task this solution is enough :)